apps/browser/bookmarks/bookmarkfiltermodel.cpp

```diff
@@ -25,12 +25,17 @@ int BookmarkFilterModel::getIndex(int currentIndex)
 }
 
 bool BookmarkFilterModel::filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const
+{
+    extendCheckPosition(sourceRow, sourceParent);
+    return ((sourceRow < m_maxPosition) && filterAcceptsRowUnbounded(sourceRow, sourceParent));
```

Comment:
Slightly counter intuitive naming as you're comparing positions as per variable names and acceptable position should be less than maxCount or <= maxPosition.

Reply:
Maybe it this should be called ```m_maxSourceModelPosition```.... Feel free to figure a better name as well.

Reply:
Thanks for your suggestion of `m_maxSourceModelPosition`; it makes good sense to me as well and I've switched to using that.